### ISSUE TITLE
find another way to warn against invalid hiccup

### ISSUE THREAD

**Issue by alidcastano** [rejoice-cljc/rectify#2]
Previously, we were wrapping component functions, and checking against invalid vector children. 

But this caused problems with react rendering, i.e. inputs would flicker on every change. 

Therefore, the following commit disabled this behavior:  
https://github.com/rejoice-cljc/rectify/commit/1e49228b824ca92bdbe10ab147238c54b1b7aa5b

I tried other approaches
- try/catch blocks when creating element -  but the error isn't triggered in this stage
- adding guards inside to component functions, actual issue is functions/symbols we don't have access to that can finalize to having invalid children

tldr; commenting out logic for now. best solution is possibly PR to react that adds custom pre-validation for children at whatever stage this happens
